packages/playout-gateway/src/coreHandler.ts

```diff
@@ -653,16 +646,19 @@ export class CoreTSRDeviceHandler {
 			)
 	}
 
-	async dispose(): Promise<void> {
+	async dispose(expected: boolean): Promise<void> {
```

Comment:
The name `expected` doesn't immediatly give me an idea of what the argument is used for.
It definetly doesn't make it obvious that it decides between `StatusCode.UNKNOWN` and `StatusCode.BAD`.

Renaming it to e.g. `expectedStatusCode`, would give an indication that the boolean effects a statuscode. Of cource simply changing the type from `boolean` to `StatusCode.UNKNOWN | StatusCode.BAD` would make it very obvivous what the expected is, and move the chose out to where the dispose is getting called.

Reply:
If you rename it to `expectedStatusCode` then the name indicates that it is a StatusCode and not a boolean.

Reply:
> If you rename it to `expectedStatusCode` then the name indicates that it is a StatusCode and not a boolean.

Was also thinking about that after i wrote my comment. Naming it `isExpectedStatusCode`, would indicate it being a boolean. The name has however grown considerably in size, so changing the type as suggested, would propably be better.